### PR TITLE
fix qnx packages definition

### PR DESCRIPTION
**Pull request by pietroalbini**

### CODE REVIEW

**Review by Hoverbear — APPROVED** [review]

File: ferrocene/packages.toml [hunk 1]
```diff
@@ -91,9 +91,9 @@ targets = [
   "x86_64-pc-nto-qnx710",
 ]
 
-[[groups.cross-compilation.packages]]
```
Comment (Hoverbear) [inline ferrocene/packages.toml:94]: Did you mean to delete this???

Comment (pietroalbini) [reply ferrocene/packages.toml:94]: Yes. With the old one, it'd add the package to the cross-compilation group, not to the qnx group.

**Comment by Hoverbear** [comment]
bors merge

**Comment by pietroalbini** [comment]
bors single on
bors p=10

**Pull request merged**